modules/tiles/src/tileset/tile-3d.ts

```diff
@@ -480,6 +493,7 @@ export class Tile3D {
     this._visibilityPlaneMask = this.visibility(frameState, parentVisibilityPlaneMask); // Use parent's plane mask to speed up visibility test
     this._visible = this._visibilityPlaneMask !== CullingVolume.MASK_OUTSIDE;
     this._inRequestVolume = this.insideViewerRequestVolume(frameState);
+    this._intersectsCullingVolume = this.intersectsCullingVolume(frameState);
```

Comment:
how does this differ from `this._inRequestVolume` is this an actual visibility check?

Reply:
ah, thanks for calling this out! This property was left over from the previous approach of dividing the tile budget evenly between tilesets intersecting the culling volume. Since it's no longer relevant to the approach we're taking now, I've removed these properties (`_intersectsCullingVolume` and `intersectsCullingVolume`).

For posterity: even if we needed to know whether a tile intersects the culling volume, we shouldn't need to add a new property, because I believe this information is already represented by the `_visible` property (exposed via the `isVisible` getter).

The confusingly-named `_inRequestVolume` property does _not_ in fact indicate whether a tile is in the camera frustum. The "requestVolume" associated with a tile is another name for its bounding volume (perhaps the "request volume" terminology comes from I3S? idk). The method `Tile3D#insideViewerRequestVolume` _sounds_ like it will indicate whether the tile is inside a "viewer request volume", but that's NOT what it does (it is very confusingly named). Instead, `Tile3D#insideViewerRequestVolume` checks whether the _camera_ is inside the tile's viewer request volume (aka bounding volume) -- a very different question.